_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "Snel.com",
+    "link": "https://snel.com/",
+    "category": "partial",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "2022.10.03",
+    "note": "Snel.com - Your friendly hosting provider"
```

The `note` field is not for this purpose.

```suggestion
    "note": ""
```